morphir/runtime/src/org/finos/morphir/runtime/MorphirRuntime.scala

```diff
@@ -11,32 +11,29 @@ import org.finos.morphir.runtime.quick.QuickMorphirRuntime
 import org.finos.morphir.runtime.MorphirRuntimeError.*
 
 trait MorphirRuntime {
-  type TypeAttribs
-  type ValueAttribs
-
   def evaluate(
-      entryPoint: Value[TypeAttribs, ValueAttribs],
-      param: Value[TypeAttribs, ValueAttribs],
-      params: Value[TypeAttribs, ValueAttribs]*
+      entryPoint: Value[scala.Unit, UType],
```

Comment:
Why are we doing this? This goes against the concept of MorphirRuntime

Reply:
So `TypeAttribs` and `ValueAttribs` here never referred to `TypedMorphirRuntimeDefs` (tho they share the same name as types defined there). As such, I think they should have been left unchanged.